sklearn/manifold/t_sne.py

```diff
@@ -645,10 +645,10 @@ class TSNE(BaseEstimator):
     """
 
     def __init__(self, n_components=2, perplexity=30.0,
-                 early_exaggeration=4.0, learning_rate=1000.0, n_iter=1000,
+                 early_exaggeration=4.0, learning_rate=200.0, n_iter=2000,
```

Comment:
Yes, maybe we can set the default to a lower value. But we should at least compare the result on the first 10,000 images from the MNIST dataset with the result from the original paper. For example, [in this case](http://nbviewer.jupyter.org/urls/gist.githubusercontent.com/AlexanderFabisch/1a0c648de22eff4a2a3e/raw/59d5bc5ed8f8bfd9ff1f7faa749d1b095aa97d5a/t-SNE.ipynb#MNIST-dataset) it works with learning_rate=1000, however, the gaps between classes in the original t-SNE paper are a little bit larger.